src/compiler/commandLineParser.ts

```diff
@@ -500,6 +500,12 @@ export const commonOptionsWithBuild: CommandLineOption[] = [
         description: Diagnostics.Set_the_language_of_the_messaging_from_TypeScript_This_does_not_affect_emit,
         defaultValueDescription: Diagnostics.Platform_specific
     },
+    {
+        name: "skipJSDocParsing",

```

Comment:
Do we want people to be able to see it? With the way we pass it in `tsc`, we force it enabled, so the user's never going to be able to disable this. Otherwise, they're in the API where it's internal and they can't see it either. So, I just said "make it never known that we chose to implement this by using compilerOptions for plumbing".

Reply:
It would also make future perf testing from the command line much easier.

Reply:
The latest version makes it possible to have as a CLI option that's overridable; do we want to make it visible?

Reply:
The latest version plumbs this differently such that it is no longer a CLI option, though we could technically make it one.